main.nf

```diff
@@ -239,8 +252,8 @@ log.info summary.collect { k,v -> "${k.padRight(21)}: $v" }.join("\n")
 log.info "===================================="
 
 // Check that Nextflow version is up to date enough
-// try / throw / catch works for NF versions < 0.25 when this was implemented
-nf_required_version = '0.28.0'
+// try / throw / catch works for NF versions < 0.30.0 when this was implemented
```

Comment:
The `nextflow.version.matches` function was still implemented in nextflow version `0.25`, that hasn't changed ;)